pkg/cmd/build/build.go

```diff
@@ -168,3 +168,16 @@ func validateImage(imageTag string) error {
 	}
 	return nil
 }
+
+func translateDockerErr(err error) error {
+	if err == nil {
+		return nil
+	}
+	if strings.HasPrefix(err.Error(), "failed to dial gRPC: cannot connect to the Docker daemon") {
+		return okErrors.UserError{
+			E:    fmt.Errorf("cannot connect to Docker Daemon"),
```

Comment:
could we add the address of the daemon? (or is this always for the local one?)

Reply:
This is just for the local docker daemon